common/src/com/thoughtworks/go/publishers/GoArtifactsManipulator.java

```diff
@@ -201,7 +201,7 @@ private String getDestPath(String file) {
     public void setProperty(JobIdentifier jobIdentifier, Property property) {
         try {
             String propertiesUrl = urlService.getPropertiesUrl(jobIdentifier, property.getKey());
```

Comment:
isn't this call in the chain good enough?
https://github.com/gocd/gocd/blob/69a39af5c0d7202b505cdf5578cebcff85643103/common/src/com/thoughtworks/go/domain/JobIdentifier.java#L213